### ISSUE TITLE
USDZ Export does not work with generated GLTF's (ie, not a pre-exisiting model)

### ISSUE THREAD

**Issue by jeffscottward** [mrdoob/three.js#22453]
**Describe the bug**

Trying to create a GLTF in the browser and convert it to USDZ without needing to download a GLTF file first.
TLDR; On-the-fly-USDZ creation and download

It successfully creates a GLTF, loads the GLTF after wrapping in blob, but fails on the USDZexporter parse.

The error I get is `Cannot read property 'uuid' of undefined`

The creation of the object is with React-Three-Fiber but its one line and would be equivalent to a similar line of code in Three.js 

This is the only relevant info I could find that might have anything to do with it
https://spectrum.chat/react-three-fiber/general/loading-gltf-getting-the-typeerror-cannot-read-property-uuid-of-undefined~a9c6f555-96da-4bdb-b21d-f1b6be783e68

**To Reproduce**

1. Go to https://rose-quokka-4btxzjoh.ws-us16.gitpod.io/
2. View model in new window (NOT THE PREVIEW)
3. Click the model
4. See how in console its looking for a UUID and can't find it

**Platform:**

 - Device: Laptop
 - OS: MacOS
 - Browser: Chrome
 - Three.js version: "0.127.0"

**Comment by mrdoob** [comment]
Please, try with a newer version of the library.
